pkg/operator/ceph/cluster/osd/deviceSet.go

```diff
@@ -75,6 +75,7 @@ func (c *Cluster) prepareStorageClassDeviceSets(config *provisionConfig) []rookv
 				Name:                storageClassDeviceSet.Name,
 				Resources:           storageClassDeviceSet.Resources,
 				Placement:           storageClassDeviceSet.Placement,
+				PreparePlacement:    storageClassDeviceSet.PreparePlacement,
```

Comment:
Can we replace `storageClassDeviceSet.PreparePlacement` by `storageClassDeviceSet.Prepare` here instead if `storageClassDeviceSet.PreparePlacement` is nil?

Reply:
I considered that, but thought it was better to preserve the spec settings as much as possible, and leave the fallback logic in the helpers when the placement is evaluated.